src/System.Text.Json/tests/Json.cs

```diff
@@ -9,6 +9,132 @@ public class Json
 
         public Array Array { get; set; }
 
+        public Value Value { get; set; }
+
+        public Json this[string index]
+        {
+            get
+            {
+                var json = new Json();
```

Comment:
NIT: this allocation will just be thrown away if we throw below...probably should move this after the throw clauses. Same with the indexer below